core/src/main/java/org/elasticsearch/action/support/replication/ReplicationResponse.java

```diff
@@ -171,7 +177,7 @@ public void writeTo(StreamOutput out) throws IOException {
 
         @Override
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-            builder.startObject(_SHARDS);
+            builder.startObject();
             builder.field(TOTAL, total);
```

Comment:
I'm also curious to understand where the rendering of the field was moved, I don't seem to find it in this PR. What is the outer object that is calling this toXContent Method?